ee/tabby-schema/src/schema/mod.rs

```diff
@@ -419,7 +423,28 @@ impl Query {
 
     async fn repository_list(ctx: &Context) -> Result<Vec<Repository>> {
         check_user(ctx).await?;
-        ctx.locator.repository().repository_list().await
+        let mut repositories = ctx.locator.repository().repository_list().await?;
+
+        repositories.extend(
+            ctx.locator
+                .config()
+                .repositories
+                .iter()
+                .enumerate()
+                .map(|(index, repo)| {
+                    Ok(Repository {
+                        id: ID::new(format!("CONFIG_{}", index)),
```

Comment:
1. the serialization / deserialization of config based id should be defined in `config.rs`
2. the format should be something like `config:${hashids encoded index}`